setup.cfg

```diff
@@ -28,7 +29,7 @@ python_requires = >=3.7
 zip_safe = False
 install_requires =
     charset_normalizer>=3.3.2
-    invenio-assets>=1.2.7
+    invenio-assets>=1.2.7,!=3.0.1
     invenio-base>=1.2.10
```

Comment:
The exclusion of invenio-assets is a short-term fix. It's important to investigate the root cause of the errors with this version and address them for future compatibility.

Reply:
Maybe @kpsherva has a better say about this.

Reply:
I think this has been fixed in v3.0.2